src/cassio/table/utils.py

```diff
@@ -19,3 +19,76 @@ def error_handler(exc: BaseException) -> None:
 
     response_future.add_callbacks(success_handler, error_handler)
     return await asyncio_future
+
+
+def handle_multicolumn_unpacking(
+    args_dict: Dict[str, Any],
+    key_name: str,
+    unpacked_keys: List[str],
+) -> Dict[str, Any]:
+    """
+    Given a dictionary of "args", handle if necessary the replacement of one of its items
+    with corresponding split-tuple version if the type structure requires it.
+    So if unpacked_keys is == [key_name], do nothing.
+    If the key_name is None, remove it from the mapping altogether.
+    Returns the modified dictionary.
+
+    Example:
+        args_dict = {"k": (1, 20), "x": "..."}
+        key_name = "k"
+        unpacked_keys = ["k_0", "k_1"]
+    results in
+        {"k_0": 1, "k_1": 20, "x": "..."}
+
+    Example:
+        args_dict = {"k": "k_val", "x": "..."}
+        key_name = "k"
+        unpacked_keys = ["k"]
+    results in (unchanged)
+        {"k": "k_val", "x": "..."}
+
+    Example:
+        args_dict = {"x": "..."}
+        args_dict = {"k": None, "x": "..."}
+    both result in
+        {"x": "..."}
+    """
+    _unp_keys = list(unpacked_keys)
+    if args_dict.get(key_name) is not None:
+        if _unp_keys != [key_name]:
+            # unpack the tuple
+            split_part = {
+                unp_k: tuple_v for unp_k, tuple_v in zip(_unp_keys, args_dict[key_name])
+            }
```

Comment:
should we do an assert before this to make sure `len(_unp_keys) == len(args_dict[key_name])` ?

Reply:
No, otherwise partial clustering specifications would not be possible

Reply:
I mean it is legitimate to have 3 clu.column but run a `get_partition` with `row_id=("a","x")` for example.

Reply:
Just an assert that len(_unp_keys) >= len(args...)` would make sense, I can add it, it does not hurt